From 302f6e537c4f67e6e0702956274935e5ac3ae959 Mon Sep 17 00:00:00 2001
From: zhenchao <zhenchao@codeaurora.org>
Date: Tue, 10 Sep 2019 19:18:17 +0800
Subject: [PATCH 05/11] Bluetooth-TWS: UI updates based on Filter matched

Change implementation of Saved Earbuds category as same as Previous Connected
Devices category for legacy devices, onProfileConnectionStateChanged in
BluetoothDeviceUpdater is used and UI updates based on Filter matched or not.

Change-Id: Iff047ccd90e81696e805fd090ada86dc0c6b2feb
CRs-Fixed: 2524220
---
 .../bluetooth/SavedBluetoothTwsDeviceUpdater.java     | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/src/com/android/settings/bluetooth/SavedBluetoothTwsDeviceUpdater.java b/src/com/android/settings/bluetooth/SavedBluetoothTwsDeviceUpdater.java
index 450431807b..d1773a9a09 100644
--- a/src/com/android/settings/bluetooth/SavedBluetoothTwsDeviceUpdater.java
+++ b/src/com/android/settings/bluetooth/SavedBluetoothTwsDeviceUpdater.java
@@ -74,17 +74,6 @@ public class SavedBluetoothTwsDeviceUpdater extends BluetoothDeviceUpdater {
         super(fragment, devicePreferenceCallback, localBluetoothManager);
     }
 
-    @Override
-    public void onProfileConnectionStateChanged(CachedBluetoothDevice cachedDevice, int state,
-            int bluetoothProfile) {
-        final BluetoothDevice device = cachedDevice.getDevice();
-        if (state == BluetoothProfile.STATE_CONNECTED) {
-            removePreference(cachedDevice);
-        } else if (state == BluetoothProfile.STATE_DISCONNECTED && device.isTwsPlusDevice()) {
-            addPreference(cachedDevice);
-        }
-    }
-
     @Override
     public boolean isFilterMatched(CachedBluetoothDevice cachedDevice) {
         final BluetoothDevice device = cachedDevice.getDevice();
-- 
2.28.0

